src/client.cpp

```diff
@@ -343,17 +343,15 @@ PVR_ERROR GetChannelStreamProperties(const PVR_CHANNEL* channel, PVR_NAMED_VALUE
 
   if (m_data && m_data->GetChannel(*channel, m_currentChannel))
   {
-    strncpy(properties[0].strName, PVR_STREAM_PROPERTY_STREAMURL, sizeof(properties[0].strName) - 1);
-    strncpy(properties[0].strValue, m_currentChannel.strStreamURL.c_str(), sizeof(properties[0].strValue) - 1);
+    PVR_STRCPY(properties[0].strName, PVR_STREAM_PROPERTY_STREAMURL);
+    PVR_STRCPY(properties[0].strValue, m_currentChannel.strStreamURL.c_str());
     *iPropertiesCount = 1;
     if (!m_currentChannel.properties.empty())
     {
       for (auto& prop : m_currentChannel.properties)
       {
-        strncpy(properties[*iPropertiesCount].strName, prop.first.c_str(),
-                sizeof(properties[*iPropertiesCount].strName) - 1);
-        strncpy(properties[*iPropertiesCount].strValue, prop.second.c_str(),
-                sizeof(properties[*iPropertiesCount].strName) - 1);
```

Comment:
here it should be `sizeof(properties[*iPropertiesCount]. strValue)`

Reply:
Good catch. Seems to be a copy / past bug.